lms/djangoapps/commerce/management/commands/tests/test_create_orders_for_old_enterprise_course_enrollmnet.py

```diff
@@ -74,7 +73,10 @@ def test_command(self, mock_create_manual_enrollment_orders):
                 output
             )
         )
-        self.assertEqual(mock_create_manual_enrollment_orders.call_count, 4)  # batch of 4 (10, 10, 10, 2)
```

Comment:
@HammadAhmadWaqas  why this assertion was not failing earlier?

Reply:
@ziafazal 
we were generating id's for the enterprise_course_enrollment with random numbers so invalid enrollments were mixed some other batches.
Now we are not generating Ids randomly so Ids are already sorted and the last 2 records will be always invalid one.